register/rxjs-all.js

```diff
@@ -0,0 +1,2 @@
+'use strict';
+require('../register')('rxjs', {Observable: require('rxjs').Observable});
```

Comment:
Should these shortcut modules export the Observable implementation they install?

```js
import {Observable} from `any-observable/register/rxjs`
```

I kinda think encouraging that use would be a mistake. We want them using this for the `--require` flag mostly.